packages/lab/src/contact-details/ContactDetails.css

```diff
@@ -43,6 +43,7 @@
 }
 
 .uitkContactDetails {
+  --contact-details-color: var(--uitk-text-primary-foreground);
```

Comment:
not necessary for this pr, but in future these need changing to --contactDetails-

Reply:
done